core/sass/partials/_effects.scss

```diff
@@ -132,18 +132,17 @@
   width: calc(100% - (#{$spacing-3 * 2}));
 
   .close {
-    background: $white url(/static/images/icons/close.png) top right no-repeat;
-    border-radius: 15px;
-    border-width: 0;
+    background-color: transparent;
+    border: none;
+    color: $active-blue;
     cursor: pointer;
-    height: 30px;
-    overflow: hidden;
-    text-indent: -1000px;
+    cursor: pointer;
+    font-size: inherit;
     padding: 0;
     position: absolute;
-    right: $spacing-4;
-    top: $spacing-4;
-    width: 30px;
+    right: 0;
+    text-decoration: underline;
+    top: 0;
```

Comment:
(can't review this specific change because it was in the previous PR) just noticed in the zeplin that the modal overlay is black with .4 opacity. does the style for .Modal-Overlay (line 157) want to be updated to match?

Reply:
Have corrected overlay.